src/base/utils/mac_utils.mm

```diff
@@ -0,0 +1,182 @@
+/*
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2015  Vladimir Golovnev <glassez@yandex.ru>
+ * Copyright (C) 2006  Christophe Dumez <chris@qbittorrent.org>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * In addition, as a special exception, the copyright holders give permission to
+ * link this program with the OpenSSL project's "OpenSSL" library (or with
+ * modified versions of it that use the same license as the "OpenSSL" library),
+ * and distribute the linked executables. You must obey the GNU General Public
+ * License in all respects for all of the code used other than "OpenSSL".  If you
+ * modify file(s), you may extend this exception to your version of the file(s),
+ * but you are not obligated to do so. If you do not wish to do so, delete this
+ * exception statement from your version.
+ */
+
+#include <CoreServices/CoreServices.h>
+#import <AppKit/AppKit.h>
+
+#include <QtGlobal>
+#if QT_VERSION >= 0x050000
+#include <QtMacExtras/QtMac>
+#endif
+
+#include "mac_utils.h"
+
+using namespace Utils::MacUtils;
+
+
+#if QT_VERSION >= 0x050000
+
+#define createQPixmapFromCGImage(cg_image)     QtMac::fromCGImageRef(cg_image)
+#define adjustQComboBoxIconSizeIfNeeded(combo) (combo)->iconSize()
+#define createQStringFromCFString(cf_string)   QString::fromCFString(cf_string)
+
+#else
+
+#define createQPixmapFromCGImage(cg_image)     QPixmap::fromMacCGImageRef(cg_image)
+static QSize adjustQComboBoxIconSizeIfNeeded(QComboBox *combo);
```

Comment:
We prefer C++ way (anonymous namespace). Fix this unless Objective-C/C++ compiler doesn't support it...